src/shared/components/case-viewer/services/case.resolver.ts

```diff
@@ -94,6 +109,10 @@ export class CaseResolver implements Resolve<CaseView> {
   private checkAuthorizationError(error: any) {
     // TODO Should be logged to remote logging infrastructure
     console.error(error);
+    if (error.status === 400 && this.requestOriginatedFrom === REQUEST_ORIGINATED_FROM) {
+      this.router. navigate(['/search/noresults'], { state: { messageId: 3 }, relativeTo: this.route });
+      return Observable.of(null);
```

Comment:
Ideally '/search/noresults' has to be public constant

Reply:
Thank you Uday, I have now made the changes as you suggested.

Reply:
@johnbenjamin-hmcts - as before, consider whether `Observable.of(...)` can be shortened to `of(...)` by using the appropriate import.